frontend/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "emeq-counter-frontend",
+  "version": "0.1.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "npx parcel index.html",
+    "build": "npx parcel build index.html",
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
The test step broke the build task. We need to write test script or remove test step from `build.gradle`.  
What do you think about that?